dts/bindings/vendor-prefixes.txt

```diff
@@ -33,6 +33,7 @@ adh	AD Holdings Plc.
 adi	Analog Devices, Inc.
 advantech	Advantech Corporation
 aeroflexgaisler	Aeroflex Gaisler AB
+aesc	aesc silicon
```

Comment:
All lower-case?

Reply:
Changed the name everywhere and capitalized it.